README.rst

```diff
@@ -0,0 +1,158 @@
+.. _README:
+
+DeepCell: Deep Learning for Single Cell Analysis
+================================================
+
+.. image:: https://travis-ci.com/vanvalenlab/deepcell-tf.svg?branch=master
+    :target: https://travis-ci.com/vanvalenlab/deepcell-tf
+    :alt: Build Status
+
+.. image:: https://coveralls.io/repos/github/vanvalenlab/deepcell-tf/badge.svg?branch=master
+    :target: https://coveralls.io/github/vanvalenlab/deepcell-tf?branch=master
+    :alt: Coverage Status
+
+.. image:: https://img.shields.io/readthedocs/deepcell?logo=Read-the-Docs
+    :target: https://deepcell.readthedocs.io/en/master/?badge=master
+    :alt: Documentation Status
+
+
+DeepCell is neural network library for single cell analysis, written in Python and built using `TensorFlow <https://github.com/tensorflow/tensorflow>`_ and `Keras <https://www.tensorflow.org/guide/keras>`_.
+
+DeepCell aids in biological analysis by automatically segmenting and classifying cells in optical microscopy images.  The framework processes raw images and uniquely annotates each cell in the image.  These annotations can be used to quantify a variety of cellular properties.
+
+Read the documentation at `deepcell.readthedocs.io <https://deepcell.readthedocs.io>`_
+
+For more information on deploying DeepCell in the cloud refer to the `DeepCell Kiosk documentation <https://deepcell-kiosk.readthedocs.io>`_.
+
+Examples
+--------
+
+.. list-table::
+
+    * - Raw Image
+      - Segmented and Tracked
+    * - .. image:: docs/images/raw.gif
+      - .. image:: docs/images/tracked.gif
+
+Getting Started
+---------------
+
+The fastest way to get started with DeepCell is to run the latest docker image:
+
+.. code-block:: bash
+
+    nvidia-docker run -it --rm -p 8888:8888 vanvalenlab/deepcell-tf:latest
+
+This will start a jupyter session, with several example notebooks detailing various training methods:
+
+For examples of how to use the deepcell library, check out the following notebooks:
+* `Training a segmentation model <https://deepcell.readthedocs.io/en/master/Training-Segmentation.html>`_
+* `Training a tracking model <https://deepcell.readthedocs.io/en/master/Training-Tracking.html>`_
+
+``deepcell.applications`` and ``deepcell.datasets``
+---------------------------------------------------------
+Together |deepcell.datasets| and |deepcell.applications|  work together to provide an accessible entrypoint to deep learning for biologists. The datasets module contains a variety of annotated datasets that can be used as training data. Additionally, the applications package initializes a set of models complete with the option to initialize with pre-trained weights.
+
+.. |deepcell.datasets| raw:: html
+
+    <tt><a href="https://deepcell.readthedocs.io/en/master/API/deepcell.datasets.html">deepcell.datasets</a></tt>
+
+.. |deepcell.applications| raw:: html
+
+    <tt><a href="https://deepcell.readthedocs.io/en/mrgn-docs/API/deepcell.applications.html">deepcell.applications</a></tt>
```

Comment:
These links have different branch names.  Is there a way to make this a relative link?

Reply:
Fixed to have both links point to master. No good options for relative links here.